package.json

```diff
@@ -54,6 +80,9 @@
     "eslint-plugin-import": "^2.6.0",
     "eslint-plugin-jsx-a11y": "^5.1.0",
     "eslint-plugin-react": "^7.1.0",
-    "react-hot-loader": "^1.3.1"
+    "react-hot-loader": "^1.3.1",
+    "jest": "^21.2.1",
+    "jest-css-modules": "^1.1.0",
```

Comment:
Do we need `jest-css-modules`? I'm not sure about it, but I think our webpack configuration is not handling css-modules. Maybe @laisvarejao can answer this.

Reply:
Jest was complaining a lot about CSS in general, so I had to add it. But if someone has a better solution, I'd gladly take it 😊